### DISCUSSION TITLE
DREAM TEXTURES: Technical Help!!!

### DISCUSSION THREAD

**Discussion by SekretVault** [community/community#37154] [Other Feature Feedback, Questions, & Ideas]
DEAR GitHub Community/Dream Textures Aficionados,

I just installed Dream Textures (as an addon) to Blender 3.2.0. (on 10/22/22).  

In "preferences" Blender acknowledges that "Model weights setup successfully." and "Addon Setup Complete".  However...

When I used Dream Textures last night for the first time (on 10/23/22), while it did indeed function "wonderfully" and correctly... when I attempted AI Upscaling the option told me:

-  Real-ESRGAN is not installed

- 1. Download the weights from GitHub

-  2. Import Model Weights.  

I successfully completed both 1. and 2. (above) in the initial download process.

As for the "Real-ESRGAN" issue:  I did find a Real-ESRGAN folder in my Blender "addons" folder on my C: Drive (which was created on 10/22/22).  So, I guessed and put the renamed "model.ckpt" file in the "weights" folder.  Was I 'sposed to do that or did I screw up?

I am so at a loss for who to contact as I cannot find any tech support or any possible way to contact anyone on this forum or on HuggingFace.

Can anyone help me with this issue?  I would seriously appreciate it.  And, I will check in with this thread, time permitting/periodically, to see if anyone replies.

I would be most grateful for any help anyone can offer me.

THANK YOU for your time and attention I really appreciate it.

**Comment by ossanna16** [comment]
Hi @SekretVault,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Dream Textures. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by SekretVault** [reply]
Hi @ossanna16,

THANK YOU for the reply and the advice.  I sure could use it as I feel like I've fallen down the proverbial "rabbit hole" on this one.  But, on a good note this has been quite the "amazing learning journey" as well.

Again, I appreciate your reply.

Sincerely,

SekretVault

**Reply by ossanna16** [reply]
That's great to hear 🙂 Keep learning and being so introspective about it. Good job!
